src/imgur/getTags.ts

```diff
@@ -40,6 +40,7 @@ export async function getTags(
   } else if (payload.slugs?.length) {
     const imagesData: Tag[] = []
     const imagePromises: Promise<Tag>[] = []
+    // TempComment
```

Comment:
This variable is updated but never used. I notice that `status` in the return is hardcoded.

❓ Can you check each of the three branches in this conditional and ensure `success` is being set how you'd like it to return based on the conditions happening. 
❓ Is there ever a case in which success would be false?

Reply:
@nonara I discussed this a bit on another comment, but this code needs to be refactored and I think this is actually a place where the testing could inform the code.

Reply:
Understood. Thanks for clarifying!